### PR TITLE
feat(history): add more lines for Teryl

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

File: src/locales/en-US/commands/history.json [hunk 1]
```diff
@@ -123,7 +123,11 @@
 	"informationTeryl": [
 		"Our oldest Archivist, from Sector 0A, guards Earth and its Solar System's colonies.",
 		"Teryl — and by extension Sector 0A — has a dedicated alert level, named **Purple Alert**, which is designed to call all the reinforcements needed to divert any enemy attacks within a single second. It was called only once, as Skyra charged to the battlefield while unleashing an attack so powerful it nearly irreversibly vaporized her heart.",
+		"Teryl has over 40000 language models from Earth. Such repository also includes historical and constructed languages.",
```
Comment (favna) [inline src/locales/en-US/commands/history.json:126]:
```suggestion
		"Teryl has over 40.000 language models from Earth. His repository also includes historical and constructed languages.",
```

File: src/locales/en-US/commands/history.json [hunk 2]
```diff
@@ -123,7 +123,11 @@
...
+		"Teryl likes to give visitors a warm welcome, offering tours around the Library and the Archive, as well as questions to pick the most suitable item specific to the user's tastes based on their answers and reactions. He often takes wild guesses when recommending, but they are also very spot on and spark the visitor's curiosity.",
```
Comment (favna) [inline src/locales/en-US/commands/history.json:127]:
```suggestion
		"Teryl likes to warmly welcome visitors, offering tours around the Library and the Archive. He also questions the visitor to pick the most suitable item specific to their tastes, based on their answers and reactions. He often takes wild guesses when recommending, but they are also very spot on and spark the visitor's curiosity.",
```

File: src/locales/en-US/commands/history.json [hunk 3]
```diff
@@ -123,7 +123,11 @@
...
 		"Teryl's body was destroyed long ago, what we see in his library is just a solid light hologram of him. He is not bothered by this, though, as it allows him to be in multiple places at the same time.",
+		"Teryl's library includes all the historical data from Earth that could be read since 2070 — the year when he started mirroring all data into his Archive — and also includes studies of his own based on related information.",
```
Comment (favna) [inline src/locales/en-US/commands/history.json:129]:
```suggestion
		"Teryl's library includes all the historical data from Earth that could be read in the year 2070 — the year when he started mirroring all data into his Archive — and also includes studies of his own based on related information.",
```

File: src/locales/en-US/commands/history.json [hunk 4]
```diff
@@ -123,7 +123,11 @@
...
+		"Teryl's library is in essence, a dream for anyone interested in books or audiobooks alike, with the Library being able to automatically translate and read out loud in virtually any language and with many different voices to pick.",
```
Comment (favna) [inline src/locales/en-US/commands/history.json:130]:
```suggestion
		"Teryl's library is in essence, a dream for anyone interested in books or audiobooks, with the Library being able to automatically translate and read out loud in virtually any language and with many different voices to pick.",
```

**Review by favna — CHANGES_REQUESTED** [review]

File: src/locales/en-US/commands/history.json (hunk 3 above)
Comment (kyranet) [reply src/locales/en-US/commands/history.json:129]: Is that right? Teryl's library includes all the reachable/readable data in the year 2070 (which includes past works) and also all the data he has collected over the millions of years he's been operative for

Comment (favna) [reply src/locales/en-US/commands/history.json:129]: that's the problem. Your version claimed that content made in 2071 would not be readable. By applying my change you do include <= 2070 and >= 2070

Comment (favna) [reply src/locales/en-US/commands/history.json]: And the sentence "the year when he started mirroring all data" already implies that it is an ongoing process.

**Review by favna — APPROVED** [review]

**Pull request merged**